### PR TITLE
Popover docs

### PR DESCRIPTION
**Pull request by svdimitr**

### CODE REVIEW

**Comment by dimodi** [comment]
Same suggestions as for the popup

File: components/popover/animation.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+* `ZoomIn`
+* `ZoomOut`
+
+See the [example](#example) below to customize the available parameters and see how they affect the Popover component.
+
+## Duration
+
+Set the `AnimationDuration` parameter in milliseconds as `int` to control how long the animation will take until the component is fully displayed.
+
+See the [example](#example) below to customize the available parameters and see how they affect the Popover component.
```
Comment (yordan-mitev) [inline components/popover/animation.md:40]:
```suggestion
The [example](#example) below lets you customize the available `AnimationDuration` parameter and see how it affects the Popover component.
```

**Review by yordan-mitev — COMMENTED** [review]
* Very good meta descriptions!
* Add the component to the navigation: 
image

File: components/popover/overview.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+| `AnimationType` | `AnimationType` enum <br /> (`SlideDown`) | Sets the type of animation when the component displays and hides. [Read more about Popover animations...]({%slug popover-animation%}) |
+| `Collision` | `PopoverCollision` enum <br /> (`Fit`) | Sets the behavior of the Popover when it doesn't fit in the viewport. [Read more about Popover collision...]({%slug popover-position-collision%}) |
+| `Offset` | `double ` | Sets the space between the Popover and its anchor in `pixels`. |
+| `Position` | `PopoverPosition  ` enum <br /> (`Top`) | Defines the position relative to the target element at which the Popover will be shown. [Read more about Popover position...]({%slug popover-position-collision%}) |
+| `ShowCallout` | `bool` <br /> (`true`) | Defines if the callout is rendered. |
+| `ShowOn` | `PopoverShowEvent` enum <br /> (`Click`) | The browser event that will display the Popover (`MouseEnter` or `Click`). When you set the ShowOn parameter to Click, the Popover will hide when the user clicks outside the component. If the parameter value is MouseEnter, the Popover will hide when the mouse pointer leaves. |
```
[the remark this answers is not in the document]
Comment (dimodi) [reply components/popover/overview.md:67]: By convention, we skip those verbs, except for boolean parameters.

[the remark this answers is not in the document]
Comment (yordan-mitev) [reply components/popover/overview.md]: I may miss the point here, but I think I see nine parameters in the table and eight of them have descriptions that start with a verb. Only one of them seems Boolean. I am still trying to figure out what makes this last parameter special.

**Review by dimodi — APPROVED** [review]

**Pull request merged**